db/db.js

```diff
@@ -209,19 +222,19 @@ async function getMostRecentSessionPhone(twilioPhone){
     }
 }
 
-async function getHistoryOfSessions(location, numEntries) {
+async function getHistoryOfSessions(locationid) {
```

Comment:
Thank you for renaming the parameter to make it clearer that it's just the ID that is being passed in.